tsconfig.json

```diff
@@ -1,13 +1,17 @@
 {
+  "ts-node": {
+    "transpileOnly": true,
+    "compiler": "ttypescript"
+  },
   "compilerOptions": {
-    "target": "ES2018",
+    "target": "ES2019",
```

Comment:
These changes (`target` and `baseUrl`) are probably unnecessary, and a result of initial attempts to make debugging support work.

Happy to revert them.